code/modules/integrated_electronics/subtypes/arithmetic.dm

```diff
@@ -236,13 +236,17 @@
 // Pi, because why the hell not? //
 /obj/item/integrated_circuit/arithmetic/pi
 	name = "pi constant circuit"
-	desc = "Not recommended for cooking. Outputs '3.14159' when it receives a pulse."
+	desc = "Not recommended for cooking. Outputs pi when it receives a pulse."
```

Comment:
Don't bother setting hit here since it'll just immediately change. alternatively just leave it like this and remove the Initialize()